functions/index.js

```diff
@@ -200,4 +204,102 @@ exports.getArcGISToken = onCall({
     logger.error('Error in getArcGISTokenR:', error);
     throw new HttpsError('internal', 'Internal server error');
   }
+});
+
+//HTTP to get ArcGIS token
+// This function is designed to be called from other backend services or functions
+// It is not intended for direct invocation from the client-side
+// It uses CORS to allow cross-origin requests, which is useful for server-to-server communication
+// The function retrieves credentials from Google Cloud Secret Manager
+exports.getArcGISTokenHttp = functions.https.onRequest(async (req, res) => {
+  // Handle CORS preflight requests and set CORS headers for actual requests.
+  cors(req, res, async () => {
+
+      logger.info("getArcGISTokenHttp function triggered.");
```

Comment:
no information in this log other than to get a count of how many times it has been called